pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -1405,3 +1436,8 @@ def load(cls, path: str) -> LiquidHandler:
 
     with open(path, "r", encoding="utf-8") as f:
       return cls.deserialize(json.load(f))
+
+
+class OperationCallback(Protocol):
+  def __call__(self, handler: "LiquidHandler", *args: Any, **kwargs: Any) -> None:
+    ...  # pragma: no cover
```

Comment:
I opted for `Protocol` because callbacks feel like their definition might expand over time. Yes, in my understanding they're similar to Swift, or Go's interfaces. `Callable` is simpler and more explicit, happy to change it.